sonar-zap-plugin/conf/webpack/webpack.config.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copy from https://github.com/SonarSource/sonar-custom-plugin-example/blob/7.x/conf/webpack/webpack.config.js
+ *
+ * Copyright (C) 2017-2017 SonarSource SA
+ * All rights reserved
+ * mailto:info AT sonarsource DOT com
+ */
+const path = require("path");
+
+module.exports = {
+  // Define the entry points here. They MUST have the same name as the page_id
+  // defined in src/main/java/org/sonarsource/plugins/example/web/MyPluginPageDefinition.java
+  entry: {
+    // Using React:
+    report_page: ["./src/main/js/report_page/index.js"],
```

Comment:
Codacy found an issue: [Identifier 'report_page' is not in camel case.](https://app.codacy.com/manual/gotimer/zap-sonar-plugin/pullRequest?prid=5861909)